lib/features/pokemon_details/pokemon_details_page.dart

```diff
@@ -33,11 +40,30 @@ class PokemonDetailsPage extends StatelessWidget {
         ),
         body: Column(
           children: <Widget>[
-            const SizedBox(
+            SizedBox(
               height: 320.0,
               child: Center(
-                /// TODO: place image, name, type, of specific pokemon here
-                child: Text(str.basicDetails),
+                /// TODO: place type of specific pokemon here
+                child: Column(
+                  children: [
+                    const SizedBox(
+                      height: 20.0,
+                    ),
+                    Text(
+                      pokemon!.name.capitalize(),
```

Comment:
This is risky. This will lead to app crash if pokemon is null. Please always handle nullable values.